_posts/en/newsletters/2021-11-17-newsletter.md

```diff
@@ -0,0 +1,154 @@
+  bech32] and another for [P2TR support][bitcoinj p2tr] to the bitcoinj repository.
+
+- **libwally-core adds bech32m support:**
+  The [0.8.4 release][libwally 0.8.4] of this wallet primative library includes [bech32m
+  support][libwally 297].
+
+- **Spark Lightning Wallet adds BOLT12 offers:**
+  Spark [v0.3.0][spark v0.3.0] adds [offer][topic offers] features including
+  offer creation, sending offer payments, and pull payments. Recurring offer
+  features are planned for a future release.
+
+- **BitGo wallets support taproot:**
+  BitGo [announced][bitgo taproot blog] support for both sending from and
+  receiving to [taproot][topic taproot] outputs using their API. Taproot support
+  in the UI is planned for a future update.
+
+- **NthKey adds bech32m send capabilities:**
+  iOS signing service [NthKey][nthkey website] added support for taproot sends
+  in the [v1.0.4 release][nthkey v1.0.4].
+
+- **Ledger Live supports taproot:**
+  Ledger's client software, Ledger Live, announced taproot support in their
+  [v2.35.0 release][ledger v2.35.0] as an experimental feature.
+
+- **Muun wallet supports taproot:**
+  Muun wallet enabled taproot address support after activation occurred,
+  including the ability for users to default to taproot receive addresses.
+
+- **Lightning-based social feed squeaknode launches:**
+  [Squeaknode][squeaknode github] is a Twitter-like P2P network based on LN
+  (using LND). Users can create, view, buy, and sell messages known as "squeaks".
+
+- **Kollider launches alpha LN-based trading platform:**
+  Kollider's latest [announcement][kollider blog] details the derivative
+  platform's features including LN deposits and withdrawals, LNAUTH and LNURL
+  support, and a LN-based referral scheme.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.14.0-beta.rc3][] is a release candidate that includes
+  additional [eclipse attack][topic eclipse attacks] protection (see
+  [Newsletter #164][news164 ping]), remote database support ([Newsletter
+  #157][news157 db]), faster pathfinding ([Newsletter #170][news170
+  path]), improvements for users of Lightning Pool ([Newsletter
+  #172][news172 pool]), and reusable [AMP][topic amp] invoices
+  ([Newsletter #173][news173 amp]) in addition to many other features
+  and bug fixes.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[BDK][bdk repo], [Bitcoin Improvement Proposals (BIPs)][bips repo], and
+[Lightning BOLTs][bolts repo].*
```

Comment:
[bolts repo] should be updated to `https://github.com/lightning/bolts` (see https://lists.linuxfoundation.org/pipermail/lightning-dev/2021-November/003337.html). Perhaps this should also be a short news item as well?

Reply:
This link is defined in `_includes/references.md` and updating it will update the links in all previous newsletters.  I think that's fine, but I don't want to do it at the last minute before the publication of this newsletter, so I'll do it in a separate PR.  We can also use that separate PR to update some related links.

Regarding a news item, I thought about it a week or two ago when it happened but decided it was mainly administrivia not development news.  However, if you or anyone else has strong feeling about it, I'm happy to add an item to next week's newsletter about it.

Reply:
I agree. Thanks Dave.